CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# V 1.2.1
```

Comment:
Why v1.2.1? The latest stable release is v1.1.0.

Reply:
I think I've gotten too used to pull requests on modules.